03_TS_events_promises/task_promises/promises.ts

```diff
@@ -0,0 +1,14 @@
+let mySymbols: string[] = ['|', '/', '-', '\\']
+
+async function loader(times: number) {
+    for(let n = 0; n < times; n++) {
+        for(let i = 0; i < mySymbols.length; i++) {
+            process.stdout.write(`\r${mySymbols[i]}`)
+            await new Promise((resolve) => setTimeout(resolve, 500));
+        }
+    }
+    process.stdout.write(`\rLoading is completed!`)
+}
+
+loader(5)
```

Comment:
Please follow the task: I should see the following output depending on the argument:
```
loader(5) => | / - \ |
loader(7) => | / - \ | / -
loader(3) => | / -
```